contracts/marketpools.js

```diff
@@ -13,17 +13,27 @@ actions.createSSC = async () => {
     const params = {};
     params.poolCreationFee = '1000';
     await api.db.insert('params', params);
+  } else {
+    const params = await api.db.findOne('params', {});
+    if (!params.poolUpdateFee) {
+      params.poolUpdateFee = '300';
```

Comment:
The mining pool update fee was recently changed to 100.  Perhaps you should change this to match?

Reply:
Replaced reward pool fee check with those from mining contract params. Maintaining separate fees for a simple wrapper seems unnecessary at this time and I want to avoid more changes to mining contract itself.

Reply:
Sounds sensible, I'm fine with this approach, thanks.